recipes/igwn-lldd-common/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "igwn-lldd-common" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://software.igwn.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: fa70188e6185e807c92fd10f2b82bcd6046435148aebda18fb3a0f8b419a6ef9
+
+build:
+  entry_points:
+    - kafka2frame = igwn_lldd_common.kafka2frame:main
+    - frame2kafka = igwn_lldd_common.frame2kafka:main
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vvv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools >=38.2.5
+  run:
+    - configargparse
+    - gpstime
+    - python >=3.6
+    - python-confluent-kafka
+    - ujson
+    - watchdog
+
+test:
+  requires:
+    - flaky
+    - kafka-python
+    - pip
+    - pytest
+  commands:
+    # check metadata
+    - python -m pip check igwn-lldd-common
+    - python -m pip show igwn-lldd-common
+    # run test suite
+    - python -m pytest -ra -v --pyargs igwn_lldd_common.tests
+    # check entry points
+    - python -m igwn_lldd_common.frame2kafka --help
+    - python -m igwn_lldd_common.kafka2frame --help
```

I think here you want to actually check the entry points as declared in `build`:

```suggestion
    - frame2kafka --help
    - kafka2frame --help
```